exporters/scss/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 <a name="1.1.0"></a>
+
```

Comment:
Please, rebase, I have fixed this here https://github.com/lmc-eu/spirit-design-system/pull/496

Reply:
I'm sorry but my branch is created from main that is up to date with origin. Are you sure @literat ]?

Reply:
OK, my mistake. It's alright now.